### DISCUSSION TITLE
Ideas for other domains to use as examples

### DISCUSSION THREAD

**Discussion by MerrionComputing** [MerrionComputing/EventsSourcing-on-Azure-Functions#50] [Q&A]
Currently the example domain is **retail banking** as it is quite well known - but any suggestions for other domains that might make an interesting demonstration of CQRS and Event Sourcing on serverless?

**Comment by DarinMacRae** [comment]
How about a messaging app?
 - Sign up
 - Topic enroll
 - Topic leave
 - Message send
 - Message read

**Reply by MerrionComputing** [reply]
Sounds interesting - is there a role for projected state (I guess things like "Topics I am currently enrolled to").
